docs/resources/vault_secrets_integration_postgres.md

```diff
@@ -0,0 +1,57 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "hcp_vault_secrets_integration_postgres Resource - terraform-provider-hcp"
+subcategory: ""
+description: |-
+  The Vault Secrets Postgres integration resource manages a Postgres integration.
+---
+
+# hcp_vault_secrets_integration_postgres (Resource)
+
+The Vault Secrets Postgres integration resource manages a Postgres integration.
+
+## Example Usage
+
+```terraform
+resource "hcp_vault_secrets_integration_postgres" "example" {
+  name         = "my-postgres-1"
+  capabilities = ["ROTATION"]
+  static_credential_details = {
+    connection_string = "postgres://user:password@localhost:5432/dbname"
+  }
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `capabilities` (Set of String) Capabilities enabled for the integration. See the Vault Secrets documentation for the list of supported capabilities per provider.
+- `name` (String) The Vault Secrets integration name.
+
+### Optional
+
+- `project_id` (String) HCP project ID that owns the HCP Vault Secrets integration. Inferred from the provider configuration if omitted.
```

Comment:
where this can be found?

Reply:
not sure what you mean (this is consistent with other docs for other providers)
see https://github.com/hashicorp/terraform-provider-hcp/pull/1143